fabric-biomes-v1/src/main/java/net/fabricmc/fabric/impl/biome/InternalBiomeData.java

```diff
@@ -134,6 +141,24 @@ public static void addNetherBiome(Biome biome, Biome.MixedNoisePoint spawnNoiseP
 		NETHER_BIOME_NOISE_POINTS.put(biome, spawnNoisePoint);
 	}
 
+	public static void addFeatureToBiome(Biome biome, GenerationStep.Feature generationStep, ConfiguredFeature<?, ?> configuredFeature) {
```

Comment:
I wish this can take a Supplier of the feature instead so I can actually just load the feature from jsons.

Reply:
+1